### ISSUE TITLE
improve co2mpas logo appearing on installation window

### ISSUE THREAD

**Issue by gfon** [JRCSTU/co2wui#118]
@stefanocorsi thank you for sharing the new beta, great work!

A first detail I noticed, the aspect ratio of the co2mpas graphic appearing on the installation window, it appears a bit slopy. Could you fix it?

![image](https://user-images.githubusercontent.com/9556847/64862697-7b311080-d633-11e9-93c7-f2039d9ae4d3.png)

**Comment by gfon** [comment]
similar also in this window, the co2mpas guy appears distorted; why don't you put them below the text so that they are not distorted
![image](https://user-images.githubusercontent.com/9556847/64862826-c2b79c80-d633-11e9-9a6f-35bbbec919ab.png)

**Comment by stefanocorsi** [comment]
Of course it can be fixed. Question: do we have some more visual material related to co2mpas? vector files of the logo, other logo flavours, etc...?

**Comment by gfon** [comment]
yes I don't remember where but @ankostis had added the logos in vectorial format on GIT
also @sapofra  had created a banner for the connected page and could provide it
for the rest I am not sure

**Comment by ankostis** [comment]
All there is from my side is the "co2mpas architecture" link, close to the top:
https://github.com/JRCSTU/co2mpas/wiki/CO2MPAS-Designated-instructions-and-material

**Comment by sapofra** [comment]
Here the banner: 
![CO2MPASS-Banner](https://user-images.githubusercontent.com/29604134/65223722-17d73080-dac2-11e9-813c-d2faccf541ca.jpg)
and the 2 avatar:
![CO2MPASS-Avatar](https://user-images.githubusercontent.com/29604134/65223843-51a83700-dac2-11e9-8e43-5a41667e57d7.png)
![CO2MPAS_Group-overview1](https://user-images.githubusercontent.com/29604134/65223988-8ddb9780-dac2-11e9-94d7-463ed67d0cce.png)

**Comment by stefanocorsi** [comment]
@gfon 

In the link provided by @ankostis or the images provided by @sapofra I see different versions of the logo. Will I use this:

![immagine](https://user-images.githubusercontent.com/6546984/65394064-29059300-dd89-11e9-8822-14fecaf382fa.png)

as the official logo?

Will I insert somewhere the EC's logo? 

![immagine](https://user-images.githubusercontent.com/6546984/65394087-8c8fc080-dd89-11e9-80e0-58d18f7aeddf.png)

What about using co2mpas result diagrams as the background for the left panel of the installation program?

![immagine](https://user-images.githubusercontent.com/6546984/65394129-ef815780-dd89-11e9-9f84-4db16d809cab.png)

Will we make a collage with some royalty free (unsplash.com) engine image as in:

![immagine](https://user-images.githubusercontent.com/6546984/65394157-4f77fe00-dd8a-11e9-85f9-e3f076ebe162.png)

Or will we use for example as a background the Co2mpas web application home page?

![immagine](https://user-images.githubusercontent.com/6546984/65394166-6a4a7280-dd8a-11e9-947a-2d4ad28c52a5.png)

**Comment by stefanocorsi** [comment]
Ok, solved the logo (not stretched anymore).

![immagine](https://user-images.githubusercontent.com/6546984/65395520-1052a880-dd9c-11e9-937a-5a4f1fb54fb7.png)

For the welcome screen of the installer I created a _temporary_ composition waiting to know what kind of visual impression we want to communicate.

![immagine](https://user-images.githubusercontent.com/6546984/65395529-4132dd80-dd9c-11e9-88d1-a4ae24a4b348.png)
